src/extension/background-script/actions/liquid/enable.ts

```diff
@@ -39,8 +47,13 @@ const enable = async (message: MessageAllowanceEnable, sender: Sender) => {
           if (!allowance.id) {
             return { data: { error: "id is missing" } };
           }
+
+          if (!enabledFor.has("liquid")) {
+            enabledFor.add("liquid");
+          }
           await db.allowances.update(allowance.id, {
             enabled: true,
+            enabledFor: [...enabledFor],
```

Comment:
why `[...enabledFor]`?

Reply:
that's a set variable name

Reply:
I don't understand. Can't you just do `enabledFor,` instead of `enabledFor: [...enabledFor],`?

Reply:
resolved. i just pushed values back in the form of an array notation. but this works as well